Sources/Brave/Frontend/Browser/Helpers/LaunchHelper.swift

```diff
@@ -127,6 +123,27 @@ public actor LaunchHelper {
   }
 }
 
+private extension FilterListStorage {
+  /// Return all the blocklist types that are valid for filter lists.
+  var validBlocklistTypes: Set<ContentBlockerManager.BlocklistType> {
+    if filterLists.isEmpty {
+      /// If we don't have filter lists yet loaded, use the settings
+      return allFilterListSettings.reduce(Set<ContentBlockerManager.BlocklistType>()) { partialResult, setting in
```

Comment:
You probably want to use the mutable variant of `reduce` for these (`reduce(into:_:)`) then use `formUnion` on the mutable versions.

Reply:
Just realized this reduce also only inserts a single item, so replace `formUnion` with `insert`